includes/admin/class-wc-admin-exporters.php

```diff
@@ -191,6 +191,30 @@ public function do_ajax_product_export() {
 			);
 		}
 	}
+
+	/**
+	 * Gets the product types that can be exported.
+	 *
+	 * @since 5.0.1
+	 * @return array The product types keys and labels.
+	 */
+	public static function get_product_types() {
+		$product_types = wc_get_product_types();
+		$product_types['variation'] = __( 'Product variations', 'woocommerce' );
+
+		/**
+		 * Allow third-parties to filter the exportable product types.
+		 *
+		 * @since 5.0.1
```

Comment:
This is going into WooCommerce 5.1 so:

```suggestion
		 * @since 5.1.0
```